R/read_params.R

```diff
@@ -46,12 +74,23 @@ read_params <- function(stage_path, return_list = FALSE) {
 #' @return absolute path to stage
 #'
 #' @export
-set_stage <- function(stage_path) {
+set_stage <- function(stage_path, quiet = F) {
```

Comment:
```suggestion
set_stage <- function(stage_path, quiet = FALSE) {
```

`T/F` is discouraged afaik and flagged by `biocCheck`